test/WebJobs.Script.Tests.Integration/ContainerManagement/LinuxContainerInitializationHostServiceTests.cs

```diff
@@ -112,43 +113,43 @@ public async Task Assigns_Context_From_CONTAINER_START_CONTEXT()
             Assert.Equal("test-key", hostSecrets.MasterKey);
         }
 
-        [Fact]
-        public async Task Assigns_Context_From_CONTAINER_START_CONTEXT_SAS_URI_If_CONTAINER_START_CONTEXT_Absent()
-        {
-            var containerEncryptionKey = TestHelpers.GenerateKeyHexString();
-            var hostAssignmentContext = GetHostAssignmentContext();
-            hostAssignmentContext.MSIContext = new MSIContext();
-            var encryptedHostAssignmentContext = GetEncryptedHostAssignmentContext(hostAssignmentContext, containerEncryptionKey);
-            var serializedContext = JsonConvert.SerializeObject(new { encryptedContext = encryptedHostAssignmentContext });
+        //[Fact]
+        //public async Task Assigns_Context_From_CONTAINER_START_CONTEXT_SAS_URI_If_CONTAINER_START_CONTEXT_Absent()
+        //{
+        //    var containerEncryptionKey = TestHelpers.GenerateKeyHexString();
+        //    var hostAssignmentContext = GetHostAssignmentContext();
+        //    hostAssignmentContext.MSIContext = new MSIContext();
+        //    var encryptedHostAssignmentContext = GetEncryptedHostAssignmentContext(hostAssignmentContext, containerEncryptionKey);
+        //    var serializedContext = JsonConvert.SerializeObject(new { encryptedContext = encryptedHostAssignmentContext });
 
-            _environment.SetEnvironmentVariable(ContainerStartContextSasUri, ContainerStartContextUri);
-            _environment.SetEnvironmentVariable(ContainerEncryptionKey, containerEncryptionKey);
-            AddLinuxConsumptionSettings(_environment);
+        //    _environment.SetEnvironmentVariable(ContainerStartContextSasUri, ContainerStartContextUri);
+        //    _environment.SetEnvironmentVariable(ContainerEncryptionKey, containerEncryptionKey);
+        //    AddLinuxConsumptionSettings(_environment);
 
-            var initializationHostService = new Mock<LinuxContainerInitializationHostService>(MockBehavior.Strict, _environment, _instanceManagerMock.Object, NullLogger<LinuxContainerInitializationHostService>.Instance, _startupContextProvider);
+        //    var initializationHostService = new Mock<AtlasContainerInitializationHostService>(MockBehavior.Strict, _environment, _instanceManagerMock.Object, NullLogger<LinuxContainerInitializationHostService>.Instance, _startupContextProvider);
+            
+        //    initializationHostService.Setup(service => service.Read(ContainerStartContextUri))
+        //        .Returns(Task.FromResult(serializedContext));
 
-            initializationHostService.Setup(service => service.Read(ContainerStartContextUri))
```

Comment:
Test not needed? if yes, please delete commented code

Reply:
Removed👍